packages/veraswap-sdk/src/query/orbiterRouters.ts

```diff
@@ -0,0 +1,35 @@
+import { queryOptions } from "@tanstack/react-query";
+import { OrbiterRouters } from "../types/OrbiterRouters.js";
+
+const orbiterRoutersMainnetUrl = "https://api.orbiter.finance/sdk/routers/v2?entry=contract";
+const orbiterRoutersTestnetUrl = "https://testnet-api.orbiter.finance/sdk/routers/v2?entry=contract";
+
+export interface OrbiterRoutersResponse {
+    status: string;
+    message: string;
+    result: OrbiterRouters[];
+}
+
+export function orbiterRoutersQueryOptions(isMainnet: boolean) {
```

Comment:
should this not be set to max cache? Or should we expect the api to update time to time?